### DISCUSSION TITLE
Audio Only Bilibili 开启仅音频模式

### DISCUSSION THREAD

**Discussion by Xinli-GUMP** [the1812/Bilibili-Evolved#4753] [功能建议 / Ideas]
**痛点:** B站有很多以`音频内容`为主的视频以及直播, 不需要视频流也不影响观看(收听)体验, 非常适合在工作、放松的时候黑听, 这时候多余的视频流不仅`浪费`大量宽带资源还影响GPU`性能`, 非常难受!

**需求:** 通过禁用视频流, 仅接收音频流的方式实现. 有个相关项目[https://github.com/cyio/audio-only-bilibili](url) 但已经很久没维护了, 相关功能失效, 也不支持直播, 希望能得以改进。

**Comment by Geeyun-JY3** [comment]
#1062、#3595
